server/src/services/sqlIngest.py

```diff
@@ -112,13 +112,55 @@ def removeInvalidClosedDates(table, report):
                 'rowsAffected': result.rowcount
             })
 
+        def setDaysToClose(table, report):
+            result = exec_sql(f"""
+              UPDATE {table}
+              SET _daystoclose = EXTRACT (
+                  EPOCH FROM
+                  (closeddate::timestamp - createddate::timestamp) /
+                  (60 * 60 * 24)
+              );
+            """)
+
+            report.append({
+              'description': 'set _daystoclose column',
+              'rowsAffected': result.rowcount
+            })
+
+        def fixNorthWestwood(table, report):
+            result = exec_sql(f"""
+              UPDATE {table}
+              SET nc = 127
+              WHERE nc = 0 AND ncname = 'NORTH WESTWOOD NC'
+            """)
+
+            report.append({
+              'description': 'fix nc code for North Westwood NC',
+              'rowsAffected': result.rowcount
+            })
+
+        def fixHistoricCulturalNorth(table, report):
+            result = exec_sql(f"""
+              UPDATE {table}
+              SET nc = 128
+              WHERE nc = 0 AND ncname = 'HISTORIC CULTURAL NORTH NC'
+            """)
+
+            report.append({
+              'description': 'fix nc code for Historic Cultural North NC',
+              'rowsAffected': result.rowcount
+            })
+
         log('\nCleaning ingest table.')
         table = Ingest.__tablename__
         report = []
 
         dropDuplicates(table, report)
         switchPrimaryKey(table, report)
         removeInvalidClosedDates(table, report)
+        setDaysToClose(table, report)
+        fixNorthWestwood(table, report)
```

Comment:
@ryanmswan This looked fine to me but i just want a sanity check before this gets merged pleassseee :pray: